plugins/inventory/linode.py

```diff
@@ -18,6 +18,8 @@
         - Uses a YAML configuration file that ends with linode.(yml|yaml).
         - Linode labels are used by default as the hostnames.
         - The inventory groups are built from groups and not tags.
+    extends_documentation_fragment:
+        - constructed
     options:
```

Comment:
```suggestion
    options:
        strict:
            version_added: 2.0.0
        compose:
            version_added: 2.0.0
        groups:
            version_added: 2.0.0
        keyed_groups:
            version_added: 2.0.0

```
(Resp. better move them to the bottom of the `options` dict, but I cannot suggest that easily in GitHub.)

Reply:
Ok.  done and pushed.